docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -32,13 +32,13 @@ az extension add --name alias
 Verify the installation of the extension with [az extension list](/cli/azure/extension#az-extension-list). If the alias extension was installed properly, it's listed in the command output.
 
 ```azurecli
-az extension list --output table
+az extension list --output table --query [].{Name:name}
```

Comment:
In some shells like `zsh`, the `[]` token has a special meaning. Use single quotes `''` around a query.